tests/test_endmembers.py

```diff
@@ -60,21 +60,23 @@ class test_endmembers(BurnManTest):
     def test0_T_ref(self):
         fo=forsterite()
         fo.params['P_0'] = 0.99999999999e5
-
+        
         fo.set_state(1.e5, 500.)
         data = [fo.H, fo.S]
         T_0 = 2000.
         fo.set_state(1.e5, T_0)
-        fo.params['T_0'] = T_0
-        fo.params['H_0'] = fo.H
-        fo.params['S_0'] = fo.S
-        fo.params['V_0'] = fo.V
-        fo.params['K_0'] = fo.K_T 
-        fo.params['Kdprime_0'] = -fo.params['Kprime_0']/fo.params['K_0']
-        fo.params['a_0'] = fo.alpha
+        fo2=fo.copy()
```

Comment:
Yes we do. The params are changed while calling these instances for the first time! We can't do this.